### ISSUE TITLE
[QoL Change] please change default tool clone directory to somewhere else other than home directory

### ISSUE THREAD

**Issue by nesper8** [rajkumardusad/onex#99]

**Comment by jetblack90** [comment]
> "please change the default clone directory to somewhere else"

I agree, I know this is a new tool and it's great, but the creator has to program it so we can download the tools to our personally organized folders instead of moving them manually every time. And the second important feature that needs to be added is to be able to download all the tools at once... there are over 400 tools and we can't just download one tool at a time.
